### PR TITLE
[qt] Select more files for translation

### LINKED ISSUE #8782: qt: Missing translations
It seems the strings in `qt/bitcoin.cpp` are not translated.

E.g. `qt/bitcoin-qt -lang=cz -datadir=/tmp/a2` gives me a warning in English that the datadir does not exist.

### PR DESCRIPTION
**Pull request by maflcko**
Fixes  #8782

### CODE REVIEW

File: src/qt/bitcoinstrings.cpp [hunk 1]
```diff
@@ -157,6 +157,9 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 QT_TRANSLATE_NOOP("bitcoin-core", ""
 "The transaction amount is too small to send after the fee has been deducted"),
 QT_TRANSLATE_NOOP("bitcoin-core", ""
+"This is a pre-release test build - use at your own risk - do not use for "
```
Comment (laanwj) [inline src/qt/bitcoinstrings.cpp]: Strange, this one is still missing - You might have to run `make translate` again to make sure these end up in bitcoin_en.ts as well.

Comment (MarcoFalke) [inline src/qt/bitcoinstrings.cpp]: Good catch! Added a commit to run `make translate`.

**Comment by sipa** [comment]
Needs rebase.

**Comment by laanwj** [comment]
utACK, but needs rebase after the other translation changes

**Comment by MarcoFalke** [comment]
Rebased

**Pull request merged**